modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/UsernameTokenValidator.java

```diff
@@ -37,92 +37,94 @@
  */
 public class UsernameTokenValidator implements Validator {
 
-	private static final int USER_SEGMENT = 0;
-	private static final int DOMAIN_SEGMENT = 1;
-	private static final String DELIMITER = "@";
-	private static Log log = LogFactory.getLog(UsernameTokenValidator.class);
+    private static final int USER_SEGMENT = 0;
+    private static final int DOMAIN_SEGMENT = 1;
+    private static final String DELIMITER = "@";
+    private static Log log = LogFactory.getLog(UsernameTokenValidator.class);
 
-	/**
-	 * This method validates the username token in SOAP message coming from the device.
-	 * @param credential  - Username token credentials coming from device
-	 * @param requestData - Request data associated with the request
-	 * @return - Credential object if authentication is success, or null if not success
-	 * @throws WSSecurityException
-	 */
-	@Override
-	public Credential validate(Credential credential, RequestData requestData) throws
-			                   WSSecurityException {
+    /**
+     * This method validates the username token in SOAP message coming from the device.
+     *
+     * @param credential  - Username token credentials coming from device
+     * @param requestData - Request data associated with the request
+     * @return - Credential object if authentication is success, or null if not success
+     * @throws WSSecurityException
```

Comment:
Be descriptive on you comments for exceptions.
ex: * @throws IllegalArgumentException if aManufacturer has no visible content.